KsApi/models/Backing.swift

```diff
@@ -6,6 +6,8 @@ public struct Backing {
   public let amount: Int
   public let backer: User?
   public let backerId: Int
+  public let completed: Bool?
+  public let backerCompleted: Bool?
```

Comment:
Let's change this to `backerCompletedAt` and make it a `Date` to match what we have on backend.